pkg/api/annotation_key_constants.go

```diff
@@ -74,6 +74,10 @@ const (
 	// This annotation can be attached to node.
 	ObjectTTLAnnotationKey string = "node.alpha.kubernetes.io/ttl"
 
+	// BootstrapCheckpointAnnotationKey represents a Resource (Pod) that should be checkpointed by
```

Comment:
Note to self: from the discussion in the proposal, we haven't decided whether an API field as well as a feature gate are needed.

Reply:
The point from the conversation was that "feature gates" are generally frowned upon if they pollute flags to components that would not otherwise use them.